priv/templates/nimble.phx.gen.template/README.md.eex

```diff
@@ -0,0 +1,69 @@
+[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+
+## Introduction
+
+> *App introduction goes here ...*
+
+## Project Setup
+
+### Erlang & Elixir
+
+* Erlang <%= erlang_version %>+ and Elixir <%= elixir_version %>+
+
+* Recommended version manager.
+
+  - [asdf](https://github.com/asdf-vm/asdf) Erlang & Elixir
```

Comment:
Controversial but I would not recommend it just yet 😛 . There is discussion about it on [Elixir Forums](https://elixirforum.com/t/elixir-version-managers/317/5):

> asdf has the big downsite, that it hides the installed system version completely and you can’t use it except by fully qualifying it (might create other problems because system and asdf executables are called in a mix) or kicking asdfs binary wrapper folder out of the $PATH

I experienced that first hand with Jetbrains-based IDE and I think it can cause issues in other cases.

Maybe let's leave this out for now and at the same time maybe we should check out [kiex](https://github.com/taylor/kiex)

Reply:
asdf work well with me :D never use [kiex](https://github.com/taylor/kiex) before, will have a look on that ;)

Reply:
I just have had a look on kiex and the first thing I saw is that doesn't support erlang, so we end up using 2 different version manage tool (1 for erlang and kiex for elixir) 🙃 .

![Pasted_Image_27_11_2020__09_47](https://user-images.githubusercontent.com/11751745/100405286-e24eaf80-3095-11eb-8220-57597814ce5f.png)

Reply:
I also found an article comparing asdf and kiex and other tools https://nts.strzibny.name/elixir-version-manager/

Reply:
Alright, I will give `asdf` a chance again 😁